main/ui/src/main/java/org/cryptomator/ui/controllers/ChangePasswordController.java

```diff
@@ -68,12 +78,29 @@ public ChangePasswordController(Application app, Localization localization) {
 	@FXML
 	private Hyperlink downloadsPageLink;
 
+	@FXML
+	private Label passwordStrengthLabel;
+
+	@FXML
+	private Rectangle passwordStrengthShape;
+
 	@Override
 	public void initialize() {
 		BooleanBinding oldPasswordIsEmpty = oldPasswordField.textProperty().isEmpty();
 		BooleanBinding newPasswordIsEmpty = newPasswordField.textProperty().isEmpty();
 		BooleanBinding passwordsDiffer = newPasswordField.textProperty().isNotEqualTo(retypePasswordField.textProperty());
 		changePasswordButton.disableProperty().bind(oldPasswordIsEmpty.or(newPasswordIsEmpty.or(passwordsDiffer)));
+		newPasswordField.textProperty().addListener((observable, oldValue, newValue) -> {
```

Comment:
Rather use `EasyBind.subscribe(newPasswordField.textProperty, this::checkPasswordStrength);`